src/zc.adoc

```diff
@@ -63,6 +63,8 @@ As C defines the same instructions as Zca, Zcf, and Zcd, the rule is that:
 * C+F implies Zcf (RV32 only)
 * C+D implies Zcd
 
+NOTE: Specifying C and Zca are equivalent on {cheri_base64_ext_name}, as Zcd is incompatible with that base architecture.
```

Comment:
Regardless, this isn't non-normative. The bullet points above need to be updated with the correct predicates, and then this text becomes redundant.